cognite/client/_api/sequences.py

```diff
@@ -901,8 +904,12 @@ def insert(
                 >>> data = c.sequences.data.retrieve(id=2,start=0,end=10)
                 >>> c.sequences.data.insert(rows=data, id=1,column_external_ids=None)
         """
+
+        columns = handle_renamed_argument(
+            columns, "columns", "column_external_ids", "insert", {"column_external_ids": column_external_ids}, False
+        )
```

Comment:
The method was written with `kwargs` in mind as it will raise if unexpected keys are present
```suggestion
        columns = handle_renamed_argument(
            columns, "columns", "column_external_ids", "insert", kwargs, False
        )
```

Reply:
Understood the intention of the `handle_renamed_argument`, however, this signature does not have a `kwargs` parameter, nor do I think we should introduce one. Which is why I set it up that way. 

How would you change it to get `kwargs`?

Reply:
Let me push a commit with what I mean